src/nvim/eval.c

```diff
@@ -8021,6 +8022,40 @@ static void f_call(typval_T *argvars, typval_T *rettv)
   (void)func_call(func, &argvars[1], selfdict, rettv);
 }
 
+// "capture(command)" function
+static void f_capture(typval_T *argvars, typval_T *rettv)
+{
+    int save_msg_silent = msg_silent;
+    garray_T *save_capture_ga = capture_ga;
+
+    if (check_secure()) {
+      return;
+    }
+
+    capture_ga = xmalloc(sizeof(garray_T));
```

Comment:
What’s the point of allocating garray here? Just create this variable in stack, in any case it is freed before function exits.

Also this should be `sizeof(*capture_ga)` even if you think allocation is needed.